src/components/NavBar.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+<div v-bind:class="{'modal-mask': (this.isFormActive)}">
+    <nav >
+        <ul><li><span>FLOWBASE</span></li></ul>
+        <ul class="navLinks">
+            <li><router-link to="/">Overview</router-link></li>
```

Comment:
Add comment regarding nested/wrapped li's needing to be changed to class-targeted css